BPRBlazor/Pages/Index.razor.cs

```diff
@@ -2,6 +2,7 @@
 using Microsoft.AspNetCore.Components;
 using Microsoft.AspNetCore.Components.Forms;
 using SevenZipExtractor;
+using BE = BPRBE.Models.Persistence;
```

Comment:
Maybe we should rename our models to be more explicit? So that we don't have to rely on prefixes to differentiate between them